working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1564,10 +1618,10 @@ Refutability of a pattern `p` matching a value of type `v` is:
 It is a compile-time error if a refutable pattern appears in an irrefutable
 context, either as the outermost pattern or a subpattern. *This means that the
 explicit predicate patterns like constants and literals can never appear in
-pattern variable declarations. The patterns that do type tests directly or
-implicitly can appear in variable declarations only if the tested type is a
-supertype of the value type. In other words, any pattern that needs to
-"downcast" to match is refutable.*
+pattern variable declarations or pattern assignments. The patterns that do type
+tests directly or implicitly can appear in variable declarations or assignments
+only if the tested type is a supertype of the value type. In other words, any
```

Comment:
This isn't true, because of dynamic?  The value type must be assignable to the value type, but not necessarily a subtype?